### PR TITLE
Add consul-ksonnet

### PR DESCRIPTION
**Pull request by gouthamve**
Moving from `deployment_tools`

### CODE REVIEW

File: consul/consul.libsonnet [hunk 1]
```diff
@@ -0,0 +1,307 @@
+{
+  _config+:: {
+    consul_replicas: 3,
+  },
+
+  _images+:: {
+    consul: 'consul:0.8.5',
+    consulSidekick: 'quay.io/weaveworks/consul-sidekick:master-f18ad13',
+    statsdExporter: 'prom/statsd-exporter:v0.6.0',
```
Comment (tomwilkie) [inline consul/consul.libsonnet]: v0.8.1 seems to be the latest.

File: consul/consul.libsonnet [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    consulExporter: 'prom/consul-exporter:v0.3.0',
```
Comment (tomwilkie) [inline consul/consul.libsonnet]: v0.4.0 seems to be the latest.

File: consul/consul.libsonnet [hunk 3]
```diff
@@ -0,0 +1,307 @@
+{
+  _config+:: {
+    consul_replicas: 3,
+  },
+
+  _images+:: {
+    consul: 'consul:0.8.5',
```
Comment (tomwilkie) [inline consul/consul.libsonnet]: v1.4.0 seems to be the latest.

**Comment by tomwilkie** [comment]
Lets use this as an opportunity to update everything.

Can you add a README too pls?

**Comment by gouthamve** [comment]
Tested by launching a single node consul cluster.

**Pull request merged**